README.md

```diff
@@ -42,7 +42,10 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+// In this particular problem it would also work to have the loop run "i <= foods.length" times, but that seems like a weak solution because it would *not* work if there was a 4th fruit. This way it steps through the entire 0th element of the array no matter how long it is.
+for (i = 0; i < foods[0].length; i++) {
+  console.log("I want to eat a " + foods[0][i]);
+}
```

Comment:
:+1:  Not sure I'm reading this right, but the foods.length actually wouldn't work in this example.. the length of foods is 2, right? and the length of foods[0] is 3.. so your code is right!